### ISSUE TITLE
Syntax error: invalid regular expressions

### ISSUE THREAD

**Issue by codytooker** [nativescript-vue/vue-cli-plugin-nativescript-vue#13]
When running the generator I get the following error in the console leading to not being able to run the generator.

```
 ERROR  SyntaxError: Invalid regular expression: /(?<fileName>[^(\s]+):[0-9]+:[0-9]+/: Invalid group
```

Seems to be related to a recent PR
https://github.com/nativescript-vue/vue-cli-plugin-nativescript-vue/pull/12

**Comment by jandever** [comment]
+1

**Comment by jawa-the-hutt** [comment]
Can I get a rundown of what options were chosen when creating the Vue project initially as well as what options were chosen when adding the plugin?  Are you using just the CLI or are you using the UI?

This will help me troubleshoot where this may be going wrong as I haven't been able to reproduce this yet with 3 different manual tests.

**Comment by jandever** [comment]
I got the same error through the CLI as with the UI. Tried multiple different options (also all default) creating the Vue project and adding the plugin. Each time the error occurs.

C:\laragon\www\vue
λ vue create vue-cli


Vue CLI v3.7.0
? Please pick a preset: default (babel, eslint)


Vue CLI v3.7.0
✨  Creating project in C:\laragon\www\vue\vue-cli.
🗃  Initializing git repository...
⚙  Installing CLI plugins. This might take a while...


> yorkie@2.0.0 install C:\laragon\www\vue\vue-cli\node_modules\yorkie
> node bin/install.js

setting up Git hooks
done

added 1158 packages from 913 contributors and audited 23634 packages in 43.246s
found 0 vulnerabilities

🚀  Invoking generators...
📦  Installing additional dependencies...

added 36 packages from 27 contributors, updated 2 packages, moved 9 packages and audited 23923 packages in 13.011s
found 0 vulnerabilities

⚓  Running completion hooks...

📄  Generating README.md...

🎉  Successfully created project vue-cli.
👉  Get started with the following commands:

 $ cd vue-cli
 $ npm run serve


C:\laragon\www\vue
λ cd vue-cli

C:\laragon\www\vue\vue-cli (master)
λ vue add vue-cli-plugin-nativescript-vue

📦  Installing vue-cli-plugin-nativescript-vue...

+ vue-cli-plugin-nativescript-vue@0.0.15
added 2 packages from 2 contributors and audited 24957 packages in 6.915s
found 0 vulnerabilities

✔  Successfully installed plugin: vue-cli-plugin-nativescript-vue

? Enter a unique application identifier: org.nativescript.application
? Use HTML5 history mode? (Default: hash mode) No
? Is this a brand new project? (Default: Yes) Yes
? Dual Native AND Web development experience or a Native only? (Default: Dual) Dual Native AND Web
? What type of template do you want to start with? (Default: Simple) Simple
adding to package.json
deleting from package.json
doing template rendering

🚀  Invoking generator for vue-cli-plugin-nativescript-vue...
 ERROR  SyntaxError: Invalid regular expression: /(?<fileName>[^(\s]+):[0-9]+:[0-9]+/: Invalid group
SyntaxError: Invalid regular expression: /(?<fileName>[^(\s]+):[0-9]+:[0-9]+/: Invalid group
    at module.exports.extractCallDir (C:\laragon\www\vue\vue-cli\node_modules\vue-cli-plugin-nativescript-vue\generator\index.js:907:22)
    at module.exports.renderDirectoryStructure (C:\laragon\www\vue\vue-cli\node_modules\vue-cli-plugin-nativescript-vue\generator\index.js:847:27)
    at api.render (C:\laragon\www\vue\vue-cli\node_modules\vue-cli-plugin-nativescript-vue\generator\index.js:199:11)
    at Generator.resolveFiles (C:\Users\JanD\AppData\Roaming\npm\node_modules\@vue\cli\lib\Generator.js:209:13)
    at <anonymous>
    at process._tickCallback (internal/process/next_tick.js:188:7)

**Comment by jawa-the-hutt** [comment]
Can I see a list of global packages installed? `npm list -g --depth=0`
Also, OS version

**Comment by jandever** [comment]
+-- @angular/cli@7.3.8
+-- @vue/cli@3.7.0
+-- @vue/cli-init@3.7.0
+-- @vue/devtools@5.1.0
+-- gulp-cli@2.0.1
+-- nativescript@5.3.4
`-- npm@6.9.0

npm ERR! peer dep missing: typescript@>=2.0, required by ts-node@8.1.0
npm ERR! peer dep missing: graphql@0.10.x - 14.1.x, required by apollo-tracing@0.5.2

**Comment by jawa-the-hutt** [comment]
What version of node are you running?

side note: realizing we need a standard 'submit issue` list of questions to ask when someone submits an issue.

**Comment by jandever** [comment]
node --version
v8.9.1

**Comment by jandever** [comment]
updating nodejs solved the problem.

**Comment by jandever** [comment]
updating nodejs solved the problem

**Comment by codytooker** [comment]
I can't currently get a rundown of global packages as I'm away from the computer but basically the same as above in installing from UI and CLI and trying multiple packages.  

I'm guessing it comes this line https://github.com/nativescript-vue/vue-cli-plugin-nativescript-vue/blob/80cc51bccdf1de4ae10155c289557351cac7a23f/generator/index.js#L908. From my knowledge only ES2018 has the ability to do look behind regular expressions.

**Comment by jawa-the-hutt** [comment]
Ok...i was able to reproduce on node version 8 regardless of the version of npm.

**Comment by jawa-the-hutt** [comment]
So, I had just commented that I was able to reproduce on any version of Node 8 and saw a couple of notifications come in at the same time with updates from @jandever and  @codytooker.  However, now the issue doesn't show those comments.

Looks like the fix is to be on node 10.x or above as node 8 doesn't support the look behinds.  I'm going to leave this issue open and see if I can come up with a regular expression that will work on node 8, 10 and 12.  

May not be possible, but at least the temporary workaround is to upgrade to at least 10.x.
